pages/orders/[orderId]/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { Button, Flex, H1, Panel, StatefulTable } from '@bigcommerce/big-design';
+import { useRouter } from 'next/router';
+import React from 'react';
+import styled from 'styled-components';
+
+import ErrorMessage from '../../../components/error';
+import Loading from '../../../components/loading';
+import { useShippingAndProductsOrder } from '../../../lib/hooks';
+import { BillingAddress, OrderProduct, ShippingAndProductsOrder } from '../../../types';
+
+const StyledAddress = styled.address`
```

Comment:
I think wherever possible, we should prefer to use BigDesign's default functionality.  Here, this largely looks similar to BD's `Text` component. We could probably add `marginVertical` to get the effect you're after with `line-height` if need be.

Reply:
I believe he is using the `<address>` element for accessibly semantics; at least that's what I would do. The rest of the styles is to clone the `Text` component.

Reply:
yeah, we're not really implementing accessibility at this point. We discussed potentially adding in an additional step or having "recipes" as Sophi would like to say. imho if someone was using a screen reader, as soon as it hits `Ship to` it would be abundantly clear what this section is. 

Let's follow-up with Booker and design about accessibility. In the meantime, this could probably be `Text`.